Documentation/Decode-Root-Keys.md

```diff
@@ -0,0 +1,28 @@
+# Decoding with Root Keys
+
+The easiest way to decode models from JSON is to use the global `decode`
+function: pass the `AnyObject` variable returned from `NSJSONSerialization`
+directly to `decode`, and get the model back. However, many times the JSON for
```

Comment:
No comma necessary here.

However, what if you say this (in which case you *would* need a comma):

"pass the `AnyObject` variable ..., and you'll get the model back"